mkdocs_awesome_pages_plugin/navigation.py

```diff
@@ -275,8 +277,8 @@ def _gather_metadata(self, items: List[NavigationItem]) -> Meta:
         paths: List[str] = []
         for item in items:
             if isinstance(item, Page):
-                if Path(self.docs_dir) in Path(item.file.abs_src_path).parents:
-                    paths.append(item.file.abs_src_path)
+                if item.file.src_path in self.files.src_paths:
```

Comment:
Yeah, it would certainly need to be a hand-crafted `File`, and even then, I don't think that's something this plugin needs to bother with (garbage in, garbage out)